vaadin-time-picker-flow/src/main/resources/META-INF/resources/frontend/timepickerConnector.js

```diff
@@ -227,13 +237,15 @@
                 };
 
                 if (previousValueObject) {
-                    const newValue = timepicker.i18n.formatTime(previousValueObject);
-                    // FIXME works but uses private API, needs fixes in web component
-                    if (timepicker.__inputElement.value !== newValue) {
-                        timepicker.__inputElement.value = newValue;
-                        timepicker.__dropdownElement.value = newValue;
-                        timepicker.__onInputChange();
-                    }
+                    when(() => timepicker.shadowRoot, () => {
```

Comment:
Wonder if `customElements.whenDefined('vaadin-time-picker').then(() => ...)` would work here instead?

Reply:
I tried it. It doesn't work in this example.

Reply:
Ok, then lgtm also